plugins/mcreator-localization/help/default/biome/mcontinentalness.md

```diff
@@ -0,0 +1,17 @@
+<html>

```

Comment:
Tooltips use MarkDown instead of HTML.So please change your tooltips for them. This way, we keep them consistent.

Reply:
This was fixed in "First Fix" (well, at least I hope so)